### DISCUSSION TITLE
Add to Cart via SKU

### DISCUSSION THREAD

**Discussion by hahnzilla** [craftcms/commerce#2398] [Ideas]
**What are you trying to do?**
We'd love to have a core way of adding an item to the cart by the sku rather than the purchasable id.

Was thinking this could be a useful feature so you don’t have to track 
around the combo of SKU/purchasableId. We have a few places where we hard 
code the SKU throughout the repo for “reasons” but we feel much better 
doing this because that SKU is unlikely to change (especially across 
environments). We try to use the core commerce add to cart because 
we’ve had issues rolling our own in the past.

**What's your proposed solution?**
Our call to the endpoint currently looks like

```
      return $axios.post('/actions/commerce/cart/update-cart', {
            qty: item.qty,
            id: item.id
      })
```

and would love the option of using the SKU

```
      return $axios.post('/actions/commerce/cart/update-cart', {
            qty: item.qty,
            sku: item.sku
      })
```

**Comment by lukeholder** [comment]
Thanks @dhahn - leaving is open as a good feature request, but not looking to add this in the short-term.
